src/pages/judge-application.js

```diff
@@ -205,6 +209,10 @@ const JudgeApplication = () => {
                         </p>
                       </div>
                     )}
+                    {isLoggedInErrorMessage ? <p 
+                    className = "error-message"
+                    > User must be logged in to submit this form. 
+                    </p> : ''}
```

Comment:
Since this page should only be interacted with by a logged-in user, we can just wrap it in a `ProtectedPage` component instead of `DefaultLayout`.  (see certified-contributor-application as an example)